preview/snap/snapcraft.yaml

```diff
@@ -2,18 +2,8 @@ name: powershell-preview
 
```

Comment:
This is the fix to resolve the seg fault.  Other changes are to support building these changes

Reply:
Would you be able to give some more information about what was causing the segfault previously and how/where these changes prevent it now?

Reply:
Can you explain what the fix is?  Is it including the additional libraries?

Reply:
To document:

- We were originally building snaps with older dependencies, but dotnet inside the snap would resolve the newest it could find on the machine, breaking snaps on systems with newer inbox versions of things like libicu
- .NET don't see this as their problem since (1) their main scenario is being the only app on the box and (2) even if they're not, they're probably expecting to be a non-classic snap
- To fix this, we now build with newer (newest?) versions of those dependencies, which is something that the snap model is essentially intended for
- However, to make this fix happen, @TravisEz13 had to do work to build the images to assemble the snap with the new dependencies, so it was more work than a simple config addition

I do think we should take this back to .NET -- it strikes me as a fairly serious issue if we fall under their support aegis, since it breaks the only reason to have a snap